week04_approx_rl/homework_tf.ipynb

```diff
@@ -92,8 +92,8 @@
     "        # Here's what you need to do:\n",
     "        #  * crop image, remove irrelevant parts\n",
     "        #  * resize image to self.img_size\n",
-    "        #     (use imresize imported above or any library you want,\n",
-    "        #      e.g. opencv, skimage, PIL, keras)\n",
+    "        #     (use resize imported above or any library you want,\n",
+    "        #      e.g. opencv, PIL, keras)\n",
```

Comment:
What's wrong with skimage? Its interface is much better than opencv's.

Reply:
> use imresize imported above

imresize is from skimage